src/DurableTask/PurgeInstancesFilter.cs

```diff
@@ -0,0 +1,13 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.DurableTask;
+
+/// <summary>
+/// Query for purging orchestration instances.
+/// </summary>
+/// <param name="CreatedFrom">Date created from.</param>
+/// <param name="CreatedTo">Date created to.</param>
+/// <param name="Statuses">The statuses.</param>
+public record PurgeInstancesFilter(
```

Comment:
I have redefined a type here because I saw that we already redefined some `DurableTask.Core` types in this package, such as `OrchestrationRuntimeStatus`. Do we want to continue down this line, or should I just use [DurableTask.Core..PurgeInstancesFilter](https://github.com/Azure/durabletask/blob/main/src/DurableTask.Core/PurgeInstanceFilter.cs) instead?

Reply:
I'm trying to avoid surfacing up DurableTask.Core types in this SDK. I want to keep it as an implementation detail just in case we decide to replace it with something different/more optimal.